src/ElevatorConfiguration.java

```diff
@@ -0,0 +1,13 @@
+/**
+ *
+ */
+
+/**
+ * The class stores shared configuration data for all subsystems.
+ *
+ * @author paulokenne
+ *
+ */
+public class ElevatorConfiguration {
```

Comment:
One of the subsystems will be called ElevatorSubsystem right? If so we should probably name this to apply to a broader context... Like SystemConfiguration? just an idea

Reply:
Also how do we expect this Class to behave? Does the controller instantiate these configs on startup? Essentially delegating the respective configs to each subsystem that gets started up

Reply:
I think we can revise how configuration works later. I think for now, I will remove the class.